### PR TITLE
Fix #348 for mongodb installation

### LINKED ISSUE #348: Fix installation guide for mongodb installation steps
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

Unable to install mongodb as it is mentioned in [guide](https://github.com/fossasia/query-server/blob/master/docs/installation/local.md).

**Expected behavior:**

Installation should succeed. As mentioned [here](https://stackoverflow.com/questions/29287915/mongodb-installation-in-ubuntu-14-04-failed) changes need to be done in guide.

**Steps to reproduce:**

Follow instructions given and try to make a fresh installation.

### PR DESCRIPTION
**Pull request by raju249**
Fixes #348 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Add correct instructions for smooth installation of mongodb.

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
@raju249 Please add changes in PR template and also please update your branch

**Comment by raju249** [comment]
Sure @vaibhavsingh97 
Doing it now.
Thanks for review.

**Comment by raju249** [comment]
@vaibhavsingh97 Updated the branch with latest master and updated PR template as well.
Thanks

**Comment by raju249** [comment]
Is @fossasia-bot not working as expected ?
Seems like its giving PR conflicts for every PR

**Comment by cclauss** [comment]
I am not a fan of @fossasia-bot

**Comment by raju249** [comment]
Yeah right 😅.
Why don't we fix it or just remove it ?
@cclauss

File: docs/installation/local.md [hunk 1]
```diff
@@ -28,13 +28,13 @@ venv\Scripts\activate # Windows
...
 
-To set up MongoDB on your server : 
+To set up MongoDB on your server :
 
 ```bash
 sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
 echo "deb http://repo.mongodb.org/apt/ubuntu "$(lsb_release -sc)"/mongodb-org/3.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.0.list
 sudo apt-get update
-sudo apt-get install -y mongodb-org
+sudo apt-get install -y mongodb
```
Comment (bhaveshAn) [inline docs/installation/local.md]: I can find that with 
`sudo apt-get install -y mongodb-org`


`E: There were unauthenticated packages and -y was used without --allow-unauthenticated` 
comes have you tried this thing

Comment (raju249) [reply docs/installation/local.md]: I didn't get this error. What I got with old version of command is the package was not located . The I checked the link mentioned on the issue page where it says to remove the org. And when I tried that while installing, it worked (removing org works )

**Comment by bhaveshAn** [comment]
Thanks @raju249

**Comment by raju249** [comment]
:+1:  :smiley:

**Pull request merged**